components/help-dropdown/help-dropdown.scss

```diff
@@ -0,0 +1,11 @@
+.custom-select {
+  display: block;
+  width: 100%;
```

Comment:
Given that this is going to be used everywhere, does 100% width make sense? Might be fine to leave now, but once we use this elsewhere it we'll probably need to reshape this a little bit.